lnrpc/rpc.proto

```diff
@@ -845,8 +845,11 @@ message EstimateFeeResponse {
     // The total fee in satoshis.
     int64 fee_sat = 1;
 
+    // Deprecated. The fee rate in satoshi/vbyte.
+    int64 feerate_sat_per_byte = 2 [deprecated = true];
+
     // The fee rate in satoshi/vbyte.
-    int64 feerate_sat_per_byte = 2;
+    uint64 feerate_sat_per_vbyte = 3;
```

Comment:
Perhaps take this opportunity to standardize `sat_per_vbyte` across all endpoints? I don't think we need the `feerate_` prefix here.